week-2/errors/1.js

```diff
@@ -4,10 +4,13 @@
 // Try playing computer with the example to work out what is going on
 
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
-  const percentage = `${decimalNumber * 100}%`;
+  // const decimalNumber = 0.5;
```

Comment:
instead of commenting out this line of code above, you should extract that line of code outside the function and then pass it to the function as a parameter. 

so you function call should look like this :
console.log (convertToPercentage(decimalNumber));